packages/consumption/test/modules/attributes/AttributesController.test.ts

```diff
@@ -1040,6 +1048,15 @@ describe("AttributesController", function () {
                 expect(updatedSuccessorChildAttribute!.succeeds).toBeUndefined();
             });
 
+            test("should detach shared attribute copies of deleted child attribute", async function () {
+                const sharedChildAttributeBeforeDeletion = await consumptionController.attributes.getLocalAttribute(predecessorChildOwnSharedIdentityAttribute.id);
+                expect(sharedChildAttributeBeforeDeletion!.shareInfo!.sourceAttribute).toStrictEqual(predecessorChildAttribute.id);
```

Comment:
The `parentId` field of the `predecessorChildOwnSharedIdentityAttribute` is already undefined before the deletion of the `predecessorComplexAttribute`, right?

Reply:
Yes